Lombiq.Analyzers.PowerShell/Rules/Measure-AutomaticVariableAlias/Measure-AutomaticVariableAlias.psm1

```diff
@@ -31,29 +31,31 @@ function Measure-AutomaticVariableAlias
         try
         {
             # Filter down tokens to just variable tokens with the name "_".
-            foreach ($automaticVariableAliasToken in $Token | Where-Object { $PSItem.GetType().Name -eq "VariableToken" -and $PSItem.Name -eq "_" })
+            foreach ($automaticVariableAliasToken in $Token | Where-Object { $PSItem.GetType().Name -eq 'VariableToken' -and $PSItem.Name -eq '_' })
             {
-                $correctionTypeName = "Microsoft.Windows.PowerShell.ScriptAnalyzer.Generic.CorrectionExtent"
+                $correctionTypeName = 'Microsoft.Windows.PowerShell.ScriptAnalyzer.Generic.CorrectionExtent'
                 $correctionExtent = New-Object -TypeName $correctionTypeName -ArgumentList @(
                     $automaticVariableAliasToken.Extent.StartLineNumber
                     $automaticVariableAliasToken.Extent.EndLineNumber
                     $automaticVariableAliasToken.Extent.StartColumnNumber
                     $automaticVariableAliasToken.Extent.EndColumnNumber
                     '$PSItem'
-                    'Replaced the usage of the alias of the automatic variable ''$_'' with its full name ''$PSItem''.'
+                    'Replaced the usage of the alias of the automatic variable "$_" with its full name "$PSItem".'
                 )
 
                 $suggestedCorrections = New-Object System.Collections.ObjectModel.Collection[$correctionTypeName]
                 $suggestedCorrections.add($correctionExtent) | Out-Null
 
                 $results += [Microsoft.Windows.Powershell.ScriptAnalyzer.Generic.DiagnosticRecord]@{
-                    "Extent" = $automaticVariableAliasToken.Extent
-                    "Message" = '''$_'' is an alias of the automatic variable ''$PSItem''. Please consider using the full name of this' +
-                    ' variable for consistency.'
-                    "RuleName" = "PSAvoidUsingAutomaticVariableAlias"
-                    "RuleSuppressionID" = "PSAvoidUsingAutomaticVariableAlias"
-                    "Severity" = "Warning"
-                    "SuggestedCorrections" = $suggestedCorrections
+                    'Extent' = $automaticVariableAliasToken.Extent
+                    'Message' = @(
+                        '"$_" is an alias of the automatic variable "$PSItem". Please use the full name of this'
+                        'variable for consistency.'
+                    ) -join ' '
```

Comment:
I find the version without `-join ' '` more readable:
```suggestion
                    'Message' = (
                        '"$_" is an alias of the automatic variable "$PSItem". Please use the full name of this' +
                        ' variable for consistency.'
                    )
```
We could even move the closing parenthesis into the previous line for more compact code:
```suggestion
                    'Message' = (
                        '"$_" is an alias of the automatic variable "$PSItem". Please use the full name of this' +
                        ' variable for consistency.')
```

I don't see the big advantage in maintenance using `-join`.

Reply:
In addition, I think the syntax without `-join` is easier to grasp for laymen, or beginners.

Reply:
Please let's discuss this and the rest in the LGHA thread (https://github.com/Lombiq/GitHub-Actions/pull/179#discussion_r1072865221), because that has all the necessary information and thought process, so we don't go back and forth.